### PR TITLE
Updates to latest Helm chart

### PR DESCRIPTION
**Pull request by gary-archer**
Remove deprecated settings from the example, eg for the configmap, and use the recommended Helm Chart settings.

### CODE REVIEW

File: 1-create-cluster.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ cd "$(dirname "${BASH_SOURCE[0]}")"
 #
 # Create or start the cluster
 #
-minikube start --cpus=2 --memory=8192 --disk-size=50g --driver=hyperkit --profile curity
+minikube start --cpus=4 --memory=16384 --disk-size=50g --driver=hyperkit --profile curity
```
Comment (mtrojanowski) [inline 1-create-cluster.sh]: Does it mean you need at least 16gb RAM to start this? 😓

File: 4-deploy-idsvr.sh [hunk 2]
```diff
@@ -74,7 +74,8 @@ fi
 helm repo add curity https://curityio.github.io/idsvr-helm 1>/dev/null
 helm repo update 1>/dev/null
 helm uninstall curity 2>/dev/null
-helm install curity curity/idsvr --values=idsvr/helm-values.yaml
+helm template curity ../idsvr-helm/idsvr --values=idsvr/helm-values.yaml > ~/Desktop/output.yaml
```
Comment (mtrojanowski) [inline 4-deploy-idsvr.sh]: Will `~/Desktop/output.yaml` work on Windows?

Not a big fan of this script creating files outside of the project's directory. Maybe at least the delete-cluster script should make a cleanup

File: idsvr/helm-values.yaml [hunk 3]
```diff
@@ -1,18 +1,18 @@
-#
-# A customised values file for the Curity Identity Server
-#
+# Default values for curity.
```
Comment (mtrojanowski) [inline idsvr/helm-values.yaml]: `curity` -> `the Curity Identity Server`

File: idsvr/helm-values.yaml [hunk 4]
```diff
@@ -102,20 +154,35 @@ ingress:
...
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+resources:
```
Comment (mtrojanowski) [inline idsvr/helm-values.yaml]: The comment now does not reflect what is in the yaml.

File: idsvr/helm-values.yaml [hunk 5]
```diff
@@ -102,20 +154,35 @@ ingress:
...
+  requests:
+    cpu: 500m
+    #memory: 128Mi
+
+autoscaling:
+  enabled: true
+  minReplicas: 2
+  maxReplicas: 4
+  targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 0
```
Comment (mtrojanowski) [inline idsvr/helm-values.yaml]: What does the `0` here mean?

**Review by mtrojanowski — CHANGES_REQUESTED** [review]
The comment near `resources` entry in `helm-values.yaml` should be updated to reflect changes, or the resource definition should be commented out.

File: 1-create-cluster.sh (hunk 1 above)
Comment (gary-archer) [reply 1-create-cluster.sh]: I suppose I was trying to say "If you are running one of the bigger installed, use more RAM". I will say this in the article instead. Changed back to 8GB.

File: 4-deploy-idsvr.sh (hunk 2 above)
Comment (gary-archer) [reply 4-deploy-idsvr.sh]: Accidental checkin - fixed

File: idsvr/helm-values.yaml (hunk 4 above)
Comment (gary-archer) [reply idsvr/helm-values.yaml]: put back default config - accidental checkin

File: idsvr/helm-values.yaml (hunk 5 above)
Comment (gary-archer) [reply idsvr/helm-values.yaml]: put back default config - accidental checkin

**Pull request merged**